dockerize.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+IMAGE_NAME="dashboard"
+DOCKER_HOME="/dashboard/"
+
+if [ "$1" == "init" ]; then
+    docker build -t $IMAGE_NAME .
+elif [ "$1" == "build" ]; then
+    docker run --rm --volume "$(pwd):$DOCKER_HOME" -ti $IMAGE_NAME yarn install --frozen-lockfile --no-bin-links
+    docker run --rm --volume "$(pwd):$DOCKER_HOME" $IMAGE_NAME npm run build
+elif [ "$1" == "run" ]; then
+    docker run --rm --volume "$(pwd):$DOCKER_HOME" -ti $IMAGE_NAME yarn install --frozen-lockfile --no-bin-links

```

Comment:
Seems odd to run the install step, then a run, but with no build in the middle. I don't have much experience with node so I couldn't say whether that's odd or not, I'll bring it up and trust your opinion.

Reply:
I believe that npm start will also build it, but in a way that enables file watchers so you can run the code & see your changes in real time. @Right2Drive  can you confirm this?